windows/CommonSettings.props

```diff
@@ -0,0 +1,94 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ImportGroup Label="PropertySheets" />
+  <PropertyGroup Label="UserMacros">
+    <BuildDir>$(SolutionDir)..\Build</BuildDir>
+    <!--NOTE: CpuOnlyBuild and UseCuDNN flags can't be set at the same time.-->
+    <CpuOnlyBuild>false</CpuOnlyBuild>
+    <UseCuDNN>true</UseCuDNN>
+    <CudaVersion>7.5</CudaVersion>
+    <!-- NOTE: If Python support is enabled, PythonDir (below) needs to be
+      set to the root of your Python installation. If your Python installation
+      does not contain debug libraries, debug build will not work. -->
+    <PythonSupport>false</PythonSupport>
+    <CudaDependencies></CudaDependencies>
+    <CudaArchitecture>compute_35,sm_35</CudaArchitecture>
```

Comment:
I would prefer to leave it as is. 
Reason for this is that CUDA compilation on Windows is **very ** slow, since CUDA compiler does everything in a single thread. This is not the case in Linux.
Adding another CUDA architecture increases build time for ~10min on my dev box.
For this reason I think that we should keep only one architecture ON by default, and everybody can change/add architecture  for their purpose.